### PR TITLE
Switch all proto structs to use strings for addresses

### LINKED ISSUE #121: Switch Addresses in proto files from bytes to string
In some of the modules' proto files, addresses are defined by bytes to be gogoproto casted to sdk.AccAddress

Example:
```
message MsgCreatePot {
  bool is_perpetual = 1; // flag to show if it's perpetual or multi-epoch
                         // distribution incentives by third party
  bytes owner = 2 [ (gogoproto.casttype) =
                        "github.com/cosmos/cosmos-sdk/types.AccAddress" ];

...
```

These should be switched to strings as this is the standard in the cosmos sdk now.

### PR DESCRIPTION
**Pull request by sunnya97**
closes #121

### CODE REVIEW

File: x/incentives/keeper/msg_server.go [hunk 1]
```diff
@@ -24,7 +24,12 @@ var _ types.MsgServer = msgServer{}
 
 func (server msgServer) CreatePot(goCtx context.Context, msg *types.MsgCreatePot) (*types.MsgCreatePotResponse, error) {
 	ctx := sdk.UnwrapSDKContext(goCtx)
-	potID, err := server.keeper.CreatePot(ctx, msg.IsPerpetual, msg.Owner, msg.Coins, msg.DistributeTo, msg.StartTime, msg.NumEpochsPaidOver)
+	owner, err := sdk.AccAddressFromBech32(msg.Owner)
```
Comment (ValarDragon) [inline x/incentives/keeper/msg_server.go:27]: Do we really have to follow this design? This is just straight up a mis-design of the SDK imo.

Why would you have all nodes on the network perform bech32 decoding in the state machine, that should just be the responsibility of the tx creator.

Comment (sunnya97) [reply x/incentives/keeper/msg_server.go:27]: Here's the context:  https://github.com/cosmos/cosmos-sdk/issues/7195

I don't fully understand it, but I'd rather not stray from what the SDK prescribes rn

Comment (ValarDragon) [reply x/incentives/keeper/msg_server.go:27]: This is pretty unfortunate.

Agreed on tabling this for now then. We have bigger impact things we can pursue for chain scaling #post-launch. This is pretty sad though =/

File: x/lockup/types/msgs.go [hunk 2]
```diff
@@ -43,11 +63,20 @@ func (m MsgBeginUnlocking) GetSignBytes() []byte {
...
 func (m MsgBeginUnlocking) GetSigners() []sdk.AccAddress {
-	return []sdk.AccAddress{m.Owner}
+	owner, _ := sdk.AccAddressFromBech32(m.Owner)
+	return []sdk.AccAddress{owner}
 }
 
 var _ sdk.Msg = &MsgUnlockTokens{}
 
+// NewMsgUnlockTokens creates a message to begin unlocking all tokens of a user
+//nolint:interfacer
```
Comment (ValarDragon) [inline x/lockup/types/msgs.go:73]: Isn't interfacer deprecated (https://github.com/mvdan/interfacer), it shouldn't be getting included in our lint list.

**Review by ValarDragon — APPROVED** [review]
LGTM once we remove nolint:interfacer

**Pull request merged**